.github/workflows/build.yml

```diff
@@ -51,3 +51,73 @@ jobs:
           use-cross: true
           command: build
           args: --release --target ${{ matrix.target }} --manifest-path bindings_ffi/Cargo.toml --target-dir bindings_ffi/target
+
+      - name: Upload binary
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ matrix.target }}.so
```

Comment:
Note that I'm using the target name as the library name so that I can disambiguate the library artifacts in the `package` step. They are renamed again when they are assembled into the zip archive.